llvm/test/CodeGen/DirectX/CreateHandle.ll

```diff
@@ -0,0 +1,61 @@
+; RUN: opt -S -dxil-op-lower %s | FileCheck %s
```

Comment:
I don't see tests for either

1. Unbounded resource arrays
2. Non-constant index into resource arrays

I think it would be good to have tests for these.

Reply:
Added a resource that does both of these things to the tests.